src/utils/swapMinAmounts.ts

```diff
@@ -109,70 +110,70 @@ export async function swapMinAmounts(
       })
   }
 
-  const allMinAmtsObj: { [key: string]: SingleCurrencyPairMinAmt } = {}
+  // Enable all swap plugins
+  for (const swapPluginConfig of Object.values(account.swapConfig)) {
```

Comment:
This can be done in parallel using Promise.all:
```
const enablePromises = Object.values(account.swapConfig).map(plugin => plugin.changeEnabled(true))
await Promise.all(enablePromises)
```

Reply:
Incorporated this comment in a separate PR.